.github/workflows/release-deploy.yml

```diff
@@ -0,0 +1,65 @@
+name: release-deploy
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: [ '3.10' ]
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: main
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Build package
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install wheel
+        cd main
+        echo "RELEASE_VERSION=$(cat pyfakefs/_version.py | cut -d "'" -f 2)" >> $GITHUB_ENV
+        python setup.py sdist bdist_wheel
```

Comment:
Modern practices suggest using `build`:

https://github.com/pytest-dev/pytest-mock/blob/b6ea52f32fd0221b0172dd46f3c8d9e04b139d35/.github/workflows/deploy.yml#L19-L25

Reply:
Thanks, that's better - will change this.